static/js/util/util_test.js

```diff
@@ -409,4 +412,24 @@ describe('utility functions', () => {
       assert.equal(formatPrice(20), "$20");
     });
   });
+
+  describe('programCourseInfo', () => {
+    it('assert program info', () => {
+      let program = findProgram(course => (
+        course.runs.length > 0 &&
+        course.runs[0].status === STATUS_PASSED
+      ));
+      const programInfoActual = programCourseInfo(program);
+      const totalCoursesExpected = program.courses.length;
+      const passedCourses = program.courses.filter(
+        course => course.runs.length > 0 && course.runs[0].status === STATUS_PASSED
+      );
+      const totalPassedCoursesExpected = passedCourses.length;
```

Comment:
i made a similar comment on a previous PR, but the approach of this test is problematic. unit tests like these should be performed with exact, understood data, and should test that the outcomes based on that data match our expectations. what you're doing here is fetching a program that matches a couple conditions, but really we don't know what the data will look like exactly. after that, you're directly copying the body of the tested function and testing your expectations against that. 

instead, you should create a program with exact data specifically for this test, pass it into the function, and make sure that the values we get back are what we expect. if you want to create the program based off of an existing one in our `DASHBOARD_RESPONSE` variable, that might work too